src/components/grid/grid.selectors.ts

```diff
@@ -0,0 +1,5 @@
+import { ApplicationState } from 'src/rootReducer'
+
+const getRoot = (state: ApplicationState) => state.grid
```

Comment:
export const getGrid = (state: ApplicationState) => state.grid

and remove export const getGrid = getRoot?

Reply:
Selectors usually follow a convention - there's a getRoot function that returns the rootState, and all the other functions continue from there (never accessing the ApplicationState directly).
Also, usually, you wouldn't use the root directly and would only get props from it. Which is not the case here, but I thought it would be nicer to establish this convention and it also makes the reducer easier to extend in the future.